lib/rules/arrow-parens.js

```diff
@@ -66,7 +66,13 @@ module.exports = {
              */
             function fixParamsWithParenthesis(fixer) {
                 const paramToken = sourceCode.getTokenAfter(firstTokenOfParam);
-                const closingParenToken = sourceCode.getTokenAfter(paramToken);
+                let closingParenToken = sourceCode.getTokenAfter(paramToken);
```

Comment:
Can we use a predicate here instead of using the if statement below? Maybe something like `sourceCode.getTokenAfter(paramToken, astUtils.isClosingParenToken)`?